shell/key-bindings.fish

```diff
@@ -1,52 +1,56 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-  # Due to a bug of fish, we cannot use command substitution,
-  # so we use temporary file instead
-  if [ -z "$TMPDIR" ]
-    set -g TMPDIR /tmp
-  end
 
-  function __fzf_escape
-    while read item
-      echo -n (echo -n "$item" | sed -E 's/([ "$~'\''([{<>})])/\\\\\\1/g')' '
+  # Use last token $cwd_esc as root for the 'find' command
+  function fzf-file-widget -d "List files and folders"
+    set -l cwd (commandline -t)
+    # The commandline token might be escaped, we need to unescape it.
+    set cwd (eval "printf '%s' $cwd")
+    if [ ! -d "$cwd" ]
+      set cwd .
```

Comment:
I still don't understand why you would need to do that: $cwd is used as a root for find, it is never passed to fzf, and thus the trailing slash should not matter.

Concretely, in the fzf project folder from the example on your screenshot, would you get different results from those two?

	command find -L shell/ -path '*/\\.*' -prune -o -type f -print -o -type d -print -o -type l -print
	command find -L shell -path '*/\\.*' -prune -o -type f -print -o -type d -print -o -type l -print

I don't.
(I've simplified the find command a bit, not that matters.)